common/persistence/sql/sqlplugin/postgres/db.go

```diff
@@ -61,17 +61,11 @@ func (pdb *db) IsDupEntryError(err error) bool {
 }
 
 func (pdb *db) IsNotFoundError(err error) bool {
-	if err == sql.ErrNoRows {
-		return true
-	}
-	return false
+	return err == sql.ErrNoRows
```

Comment:
since we're working on wrapped errors: are you on a recent master?  I'm not sure what the sequencing of these changes is.  since this'd be a fairly obvious `errors.Is(err, sql.ErrNoRows)` transformation.
(but probably not worth doing in this PR, keep this one full of obviously-noops)

Reply:
This is on recent master.
Yeah, lets leave it a separate PR, that addresses such cases.